app/src/main/java/io/homeassistant/companion/android/sensors/NotificationSensorManager.kt

```diff
@@ -184,4 +195,58 @@ class NotificationSensorManager : NotificationListenerService(), SensorManager {
             Log.e(TAG, "Unable to update active notifications", e)
         }
     }
+
+    private fun updateMediaSession(context: Context) {
+        if (!isEnabled(context, mediaSession.id))
+            return
+
+        val mediaSessionManager = context.getSystemService(Context.MEDIA_SESSION_SERVICE) as MediaSessionManager
+        val mediaList = mediaSessionManager.getActiveSessions(ComponentName(context, NotificationSensorManager::class.java))
+        val sessionCount = mediaList.size
+        var primaryTitle = mediaList[0].metadata?.getString(MediaMetadata.METADATA_KEY_TITLE)
+            ?: if (sessionCount > 0) getPlaybackState(mediaList[0].playbackState?.state) else "Unavailable"
+        if (sessionCount > 0 && getPlaybackState(mediaList[0].playbackState?.state) != "Playing")
+            primaryTitle = getPlaybackState(mediaList[0].playbackState?.state)
+        val attr: MutableMap<String, Any?> = mutableMapOf()
+        val icon = "mdi:play-circle"
+        if (mediaList.size > 0) {
+            for (item in mediaList) {
+                attr += mapOf(
+                    "artist_" + item.packageName to item.metadata?.getString(MediaMetadata.METADATA_KEY_ARTIST),
+                    "album_" + item.packageName to item.metadata?.getString(MediaMetadata.METADATA_KEY_ALBUM),
+                    "title_" + item.packageName to item.metadata?.getString(MediaMetadata.METADATA_KEY_TITLE),
+                    "duration_" + item.packageName to item.metadata?.getLong(MediaMetadata.METADATA_KEY_DURATION),
+                    "media_id_" + item.packageName to item.metadata?.getString(MediaMetadata.METADATA_KEY_MEDIA_ID),
+                    "playback_position_" + item.packageName to item.playbackState?.position,
+                    "playback_state_" + item.packageName to getPlaybackState(item.playbackState?.state)
+                )
+            }
+        }
+        attr += mapOf("total_media_session_count" to sessionCount)
+        onSensorUpdated(
+            context,
+            mediaSession,
+            primaryTitle,
```

Comment:
I think state should be the playback state, not potentially the title.  That way it stays consistent.